javascripts/discourse/templates/components/dc-collective-alert.hbs

```diff
@@ -0,0 +1,13 @@
+{{#unless hidden}}
+<div class="row">
+  <div class="alert alert-info alert-collective-alert">
+    <span>
+      {{{message}}}
+    </span>
+    {{d-button action=(action "join")
+    class="btn-primary"
+    label="discourse_debtcollective_collectives.join"
```

Comment:
Are you sure translations works here like that? the way we have been using this is like:
`I18n.t(themePrefix("dc.popup.description"))` where `dc.popup.description` is the label